packages/block-library/src/cover/style.scss

```diff
@@ -63,16 +49,6 @@
 		opacity: 0.5;
 	}
 
-
-	@for $i from 1 through 10 {
-		&.has-background-dim.has-background-dim-#{ $i * 10 } {
```

Comment:
We can not delete these classes, otherwise, for current blocks using these classes during the next WordPress update the classes will not be present and the blocks visual will change on the frontend.